games-arcade/rocksndiamonds/additional-files/rocksndiamonds.rdef

```diff
@@ -0,0 +1,39 @@
+// Rocks 'n' Diamonds icon by Stephanie Fu
+
+resource app_signature "application/x-vnd.rocksndiamonds";
+
+resource app_flags B_EXCLUSIVE_LAUNCH;
+
+resource app_version {
+	major = 0,
+	middle = 0,
+	minor = 0,
```

Comment:
Yep, I was a little bit puzzled, because I could not find your comments ;-)
BTW, when I started writing my review your message on GCI was not there yet; And I only saw it after I submitted my message about my review. So this was a big coincidence!

@dash102; as you can see, we are all happy mentors!

@Begasus, how come did I forget about #745 !
Maybe we could create a task for providing feedback to your PR and make it build 4.0.0.0 or fixing the remaining issues in your recipe and updating it to 4.0.0.0.
Well, this is just a suggestion. I'll let you create the task if you wish ;-)